packages/zowe-explorer/src/Profiles.ts

```diff
@@ -557,6 +572,50 @@ export class Profiles extends ProfilesCache {
         return profileType;
     }
 
+    public async createZoweSchema(zoweFileProvider: IZoweTree<IZoweTreeNode>) {
+        try {
+            ImperativeConfig.instance.loadedConfig = {
+                defaultHome: path.join(os.homedir(), ".zowe"),
+                envVariablePrefix: "ZOWE",
+            };
+
+            // If there are no workspaces already opened, default to the cliHome
+            const rootPath = vscode.workspace.workspaceFolders?.[0].uri.fsPath || ImperativeConfig.instance.cliHome;
+            // true-case-path may or may not be needed. Better safe than sorry : )
+            const config = await Config.load("zowe", { projectDir: trueCasePathSync(rootPath) });
+
+            const impConfig: IImperativeConfig = zowe.getImperativeConfig();
+            const knownCliConfig: ICommandProfileTypeConfiguration[] = impConfig.profiles;
+            knownCliConfig.push(impConfig.baseProfile);
+            config.setSchema(ConfigSchema.buildSchema(knownCliConfig));
+
+            // Note: IConfigBuilderOpts not exported
+            // const opts: IConfigBuilderOpts = {
+            const opts: any = {
+                // getSecureValue: this.promptForProp.bind(this),
+                populateProperties: true,
+            };
+
+            // Build new config and merge with existing layer
+            const newConfig: IConfig = await ConfigBuilder.build(impConfig, opts);
+            config.api.layers.merge(newConfig);
+            await config.save(false);
+            const reloadButton = localize("createZoweSchema.reload.button", "Reload Window");
+            const infoMsg = localize(
+                "createZoweSchema.reload.infoMessage",
+                "Team Configuration file created. Location: {0}. \n Please reload your window.",
+                rootPath
+            );
+            await vscode.window.showInformationMessage(infoMsg, ...[reloadButton]).then(async (selection) => {
+                if (selection === reloadButton) {
+                    await vscode.commands.executeCommand("workbench.action.reloadWindow");
+                }
+            });
```

Comment:
Is there a way we can re-read the config files without having the user reload vsc?  Having this will also be a good fix for the refresh button in the views since they aren't working for config files either and user has to reload vsc with every change to the config file.

Reply:
@JillieBeanSim I think I would like to handle this in a separate PR and include the reload of profiles even for the create, update, and delete of profiles in the config file. I can file an issue to track this. What do you think?

Reply:
@crawr I think a separate PR would be fine, not sure if create, update, and delete will work though since it just opens the config file but having it work for this and view's refresh button would be a great start.  That way after create, update, and delete actions are done by the user on the config file they could just click the refresh button and those changes will be seen without restarting VSC.

Reply:
#1514 created to track the research for this